### PR TITLE
run cpp with dockerfile

### PR DESCRIPTION
**Pull request by cdetrio**

### CODE REVIEW

**Comment by axic** [comment]
Why is there a `Dockerfile` and a `hera.dockerfile` here?

File: ewasm-testnet-cpp-config.json [hunk 1]
```diff
@@ -38,6 +38,26 @@
...
+    },
+    "1000000000000000000000000000000000000000" : {
+      "balance" : "0x00",
+      "code" : "0x0061736d01000000010f0360037f7f7f0060027f7f0060000002310208657468657265756d0c63616c6c44617461436f7079000008657468657265756d0c73746f7261676553746f72650001030201020503010001071102046d61696e0002066d656d6f727902000a1201100041204100410210004100412010010b",
+      "nonce" : "0x00",
+      "storage" : {
+        "0x99": "0x99"
+      }
+    },
+    "1000000000000000000000000000000000000001" : {
```
Comment (axic) [inline ewasm-testnet-cpp-config.json]: These three entires should be removed before merge.

File: hera.dockerfile [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        gmp \
+        libcurl \
+        libmicrohttpd
+
+#RUN git clone --recursive https://github.com/ethereum/cpp-ethereum --branch develop --single-branch --depth 1
+RUN git clone https://github.com/ethereum/cpp-ethereum --branch develop --single-branch
+# pin cpp-ethereum to commit ee0c6776..
+RUN cd cpp-ethereum && git reset --hard ee0c6776c01b09045a379220c7e490000dae9377
+RUN cd cpp-ethereum && git submodule update --init
+RUN cd cpp-ethereum/hera && git pull origin master
```
Comment (axic) [inline hera.dockerfile]: Should just get the `ewasm` branch off cpp-eth.

**Comment by jwasinger** [comment]
@cdetrio @axic I've updated the dockerfile and removed the `hera.dockerfile` (it's not used afaik).

**Comment by lrettig** [comment]
From https://github.com/ewasm/testnet/issues/4: Build a docker image of a client (with [rpcproxy/dopple](https://github.com/ethereum/dopple) for easy testing)

File: ewasm-testnet-cpp-config.json [hunk 3]
```diff
@@ -1,44 +1,44 @@
...
-    "0000000000000000000000000000000000000002": { "precompiled": { "name": "sha256", "linear": { "base": 60, "word": 12 } }, "balance": "0x01" },
-    "0000000000000000000000000000000000000003": { "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } }, "balance": "0x01" },
-    "0000000000000000000000000000000000000004": { "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } }, "balance": "0x01" },
-    "0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp" }, "balance": "0x01" },
-    "0000000000000000000000000000000000000006": { "precompiled": { "name": "alt_bn128_G1_add", "linear": { "base": 500, "word": 0 } }, "balance": "0x01" },
-    "0000000000000000000000000000000000000007": { "precompiled": { "name": "alt_bn128_G1_mul", "linear": { "base": 40000, "word": 0 } }, "balance": "0x01" },
-    "0000000000000000000000000000000000000008": { "precompiled": { "name": "alt_bn128_pairing_product" }, "balance": "0x01" },
-    "000000000000000000000000000000000000000a": { "codeFromFile": "metering.wasm", "balance": "0x01" }
```
Comment (axic) [inline ewasm-testnet-cpp-config.json]: This was removed and as such "should be breaking proper Hera". Any reason for removing this?

**Pull request merged**